content/200-orm/300-prisma-migrate/300-workflows/10-seeding.mdx

```diff
@@ -344,6 +346,101 @@ psql file.sql
 
 </TabbedContent>
 
+### Seeding your database with <inlinecode>@snaplet/seed</inlinecode>
+
+[`@snaplet/seed`](https://docs.snaplet.dev/seed/getting-started/overview) offers a toolkit designed to understand your database schema,
```

Comment:
we might want to use the "type-safe" expression somewhere in there. It's appealing for Prisma users.

Reply:
Added something toward that.